src/usr/local/www/system_usermanager_settings.php

```diff
@@ -221,7 +226,7 @@
 	'*Authentication Server',
 	$pconfig['authmode'],
 	$auth_servers
-));
+))->setHelp('To allow login with RADIUS credentials, equivalent local users with the expected privileges must be created first.'); 
```

Comment:
This note should specifically call out SSH. That isn't needed for GUI logins, just SSH.

Reply:
SSH and console (pam system)